cmd/rekor-cli/app/pflags.go

```diff
@@ -79,6 +79,7 @@ func addArtifactPFlags(cmd *cobra.Command) error {
 	cmd.Flags().Var(&fileOrURLFlag{}, "public-key", "path or URL to public key file")
 
 	cmd.Flags().Var(&fileOrURLFlag{}, "artifact", "path or URL to artifact file")
+	cmd.Flags().String("artifact-hash", "", "hex encdoed hash of artifact (when using URL)")
```

Comment:
I'd change this to be of type `uuidFlag` so you get the checking for hex string of 64 char long; also maybe put that it must be SHA256 in the help text?

Reply:
Nice,done.